op-program/client/interop/consolidate.go

```diff
@@ -180,7 +181,8 @@ func checkHazards(
 		}
 	}
 
-	hazards, err := cross.CrossUnsafeHazards(deps, chainID, candidate, execMsgs)
+	hazardDeps := &cross.UnsafeHazardDeps{UnsafeStartDeps: deps}
```

Comment:
this adapter isn't needed. As `cross.CrossUnsafeHazards` already wraps the `UnsafeStartDeps` input as `CrossUnsafeHazards`.